### PR TITLE
Still gather bootstrap if no master IPs are gathered

### PR DESCRIPTION
**Pull request by clacroix12**
Jira: [OCSQE-254](https://projects.engineering.redhat.com/browse/OCSQE-254)

Signed-off-by: Coady LaCroix <clacroix@redhat.com>

### CODE REVIEW

File: ocs_ci/utility/bootstrap.py [hunk 1]
```diff
@@ -43,12 +43,10 @@ def gather_bootstrap():
         f"--dir {gather_bootstrap_dir} --log-level debug --key {ssh_key} "
     )
     if len(master_ips) == 0:
-        raise NodeNotFoundError(
-            "No master nodes found for cluster, "
-            "unable to gather bootstrap data"
-        )
-    for master in master_ips:
-        cmd += f"--master {master} "
+        cmd += "--master None "
```
Comment (petr-balogh) [inline ocs_ci/utility/bootstrap.py:46]: I would write some warning message here.

Comment (clacroix12) [reply ocs_ci/utility/bootstrap.py:50]: Done!

**Comment by petr-balogh** [comment]
Is this verified @clacroix12 ?

**Comment by clacroix12** [comment]
@petr-balogh PR deployment passed. I manually tested this logic with an empty list of IPs as well since the scenario in which this code executes is not easily reproducible on command.

**Pull request merged**